### ISSUE TITLE
Daily note doesn't work since update to 0.21.3 and 0.21.2

### ISSUE THREAD

**Issue by nteike** [foambubble/foam#1190]
### Describe the bug

Since updates to version 21.3 and 22.2 the foam command "Open daily note" does not work any more. It still shows the menu to select the date, but on selection it does not open or create a note. When rolling back to version 21.1 everything works fine.

### Steps to Reproduce the Bug or Issue

1. Press alt+d or alt+h
2. In case of alt+h: select date of note to open/create
3. Nothing happens.

### Expected behavior

I expected, that on selection of date or pressing alt+d a daily note would open or be created, as it did in prior versions.

### Operating System Version

Windows 10

### Visual Studio Code Version

Version: 1.77.3 (user setup) Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710 Date: 2023-04-12T09:16:02.548Z Electron: 19.1.11 Chromium: 102.0.5005.196 Node.js: 16.14.2 V8: 10.2.154.26-electron.0 OS: Windows_NT x64 10.0.19044 Sandboxed: No

**Comment by nteike** [comment]
I attached a ZIP archive of a VS Code workspace. Even I deleted all notes, the git repo and all other personal stuff, it still wont create/open daily notes. [Notizen.zip](https://github.com/foambubble/foam/files/11230481/Notizen.zip) 

You're right, it seems like a duplicate of #1188

**Comment by AlogentCBrink** [comment]
I am running v0.21.3 and am having the same problem.
When I select any date to create a new Daily Note, There is nothing in the FOAM output.
```
Foam Logging: info
[info - 8:19:13 AM] Starting Foam
[info - 8:19:13 AM] Loading from directories:
[info - 8:19:13 AM] - c:\Brinks Stuff\repos\Brink\foam
[info - 8:19:13 AM]   Include: **/*
[info - 8:19:13 AM]   Exclude: **/.vscode/**/*,**/_layouts/**/*,**/_site/**/*,**/node_modules/**/*,**/.git,**/.svn,**/.hg,**/CVS,**/.DS_Store,**/Thumbs.db,**/node_modules
[info - 8:19:14 AM] Workspace loaded in 390ms
[info - 8:19:14 AM] Graph loaded in 6ms
[info - 8:19:14 AM] Tags loaded in 0ms
[info - 8:19:14 AM] Loaded 86 resources
```

**Comment by riccardoferretti** [comment]
This should now be fixed in 0.21.4
For any further comment let's use #1188
